hangman.c

```diff
@@ -0,0 +1,175 @@
+/*Sean Kee*/
+/*Hangman v1.5*/
+#include <stdio.h>
```

Comment:
Couple of things: 
-the program initializes it with a head already! 
-you aren't doing the comparisons with the guesses correctly, which leads to the user never being able to correctly guess
-nice hangman art! 
though you didn't get it properly working, nice progress on it